corrcal/test/test_corrcal_fof.py

```diff
@@ -1,67 +1,76 @@
-import numpy
+import numpy as np
+from scipy.optimize import fmin_cg
+
 from corrcal import corrcal
```

Comment:
There seem to be issues with the wrapper module.
It seems that the corrcal c library is not where we think it is?

Traceback (most recent call last):
  File "test_corrcal_fof.py", line 4, in <module>
    from corrcal import corrcal
  File "/home/ronniyjoseph/Software/anaconda3/lib/python3.7/site-packages/corrcal-0.1.dev76+g4c92827-py3.7-linux-x86_64.egg/corrcal/__init__.py", line 1, in <module>
    from .corrcal import *
  File "/home/ronniyjoseph/Software/anaconda3/lib/python3.7/site-packages/corrcal-0.1.dev76+g4c92827-py3.7-linux-x86_64.egg/corrcal/corrcal.py", line 5, in <module>
    from . import wrapper
  File "/home/ronniyjoseph/Software/anaconda3/lib/python3.7/site-packages/corrcal-0.1.dev76+g4c92827-py3.7-linux-x86_64.egg/corrcal/wrapper.py", line 21, in <module>
    LIB_LOC = glob(site.getsitepackages()[0] + "/corrcal/c_corrcal*.so")[0]
IndexError: list index out of range

Reply:
After uninstalling and reinstalling, issue has been resolved